src/platform.js

```diff
@@ -41,12 +41,12 @@ exports = module.exports = {
 	* `true` if the platform has native single-finger [events]{@glossary event}.
 	* @public
 	*/
-	touch: Boolean(('ontouchstart' in window) || window.navigator.msMaxTouchPoints),
+	touch: Boolean(('ontouchstart' in window) || window.navigator.msMaxTouchPoints || window.navigator.maxTouchPoints),
```

Comment:
I agree that we should probably be checking for >1 below for gesture events, but we shouldn't be checking for *exactly* 1 here - as we need to check for the existence of at least 1 touch point. It's likely that maxTouchPoints will return more than 1.